### PR TITLE
#166272744 set up and deploy to heroku

### PR DESCRIPTION
**Pull request by Philipotieno**
#### What does this PR do?
 -  Set up and deploy app to heroku
#### Description of Task to be completed?
 - Build a simple express server to serve the application. 
-  Add scripts to start the server and run app on Heroku
#### How should this be manually tested?
- `Staging`: [https://ah-immortals-frontend-staging.herokuapp.com/](https://ah-immortals-frontend-staging.herokuapp.com/)
-  `Production`: [https://ah-immortals-frontend.herokuapp.com/](https://ah-immortals-frontend.herokuapp.com)
#### Screenshots
image

#### What are the relevant pivotal tracker stories?
[#166272744](https://www.pivotaltracker.com/n/projects/2326460/stories/166272744)

### CODE REVIEW

**Review by IssaIan — CHANGES_REQUESTED** [review]
@Philipotieno could you update the readme.md to have the link to the app on heroku accordingly?

**Review by IssaIan — APPROVED** [review]

**Comment by a-braham** [comment]
hello @Philipotieno would you resolve the conflicts

**Review by a-braham — APPROVED** [review]
LGTM

**Comment by Philipotieno** [comment]
@a-braham I have resolved the conflicts

**Review by Amoswachira — APPROVED** [review]

**Review by matthenge — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "test:watch": "jest --watch",
     "build": "webpack -p ",
     "start:dev": "webpack -d && webpack-dev-server --mode development --open --hot",
-    "lint": "./node_modules/.bin/eslint"
+    "lint": "./node_modules/.bin/eslint",
+    "start": "node server.js",
+    "heroku-postbuild": "webpack -p"
```
Comment (koechkevin) [inline package.json]: @Philipotieno this command is redundant compare with build command on line 9. so it means heroku runs `webpack -p` twice

File: package.json [hunk 2]
```diff
@@ -8,7 +8,9 @@
     "test:watch": "jest --watch",
     "build": "webpack -p ",
```
Comment (koechkevin) [inline package.json]: @Philipotieno just remove the space after the command

**Review by koechkevin — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Philipotieno) [reply package.json]: Ok, it's done @koechkevin

**Review by koechkevin — APPROVED** [review]

**Pull request merged**